app/src/main/java/com/example/pps_tudai/mvp/presenter/StepsCounterPresenter.java

```diff
@@ -0,0 +1,72 @@
+package com.example.pps_tudai.mvp.presenter;
+
+import android.hardware.Sensor;
+import android.hardware.SensorEvent;
+import android.hardware.SensorEventListener;
+import android.hardware.SensorManager;
+
+import com.example.pps_tudai.data.entities.entity.User;
+import com.example.pps_tudai.mvp.model.StepsCounterModel;
+import com.example.pps_tudai.mvp.view.StepsCounterView;
+
+import static com.example.pps_tudai.utils.IntUtils.ZERO;
+import static com.example.pps_tudai.utils.StringUtils.FALSE;
+import static com.example.pps_tudai.utils.StringUtils.TRUE;
+
+public class StepsCounterPresenter implements SensorEventListener {
+
+    private static final int ONE = 1;
+    private final StepsCounterView counterView;
+    private final StepsCounterModel counterModel;
+    private User user;
+    SensorManager sensorManager;
+    Sensor sensor;
```

Comment:
sensorManager & sensor should be private

Reply:
fixed it